src/Compute/Compute/VirtualMachine/Operation/NewAzureVMCommand.cs

```diff
@@ -770,6 +770,11 @@ async Task StrategyExecuteCmdletAsync(IAsyncCmdlet asyncCmdlet)
             PublicIpAddressName = PublicIpAddressName;
             SecurityGroupName = SecurityGroupName ?? Name;
 
+            if (this.IsParameterBound(c => c.Image))
+            {
+                WriteInformation(HelpMessages.PSVMImageMessage, new string[] { "PSHOST" });
```

Comment:
Why should this be written whenever the user is already using the Image parameter? Though I guess this is the best option since we don't want to maintain a hardcoded list of images. Ok, this works for a client side implementation.